sympy/printing/precedence.py

```diff
@@ -129,6 +129,9 @@ def precedence(item):
                 return PRECEDENCE_FUNCTIONS[n](item)
             elif n in PRECEDENCE_VALUES:
                 return PRECEDENCE_VALUES[n]
+    # Check if it's a negative number, and add parentheses
+    if isinstance(item, (int, float, complex)) and item < 0:
```

Comment:
Would these raw Python types be the value here or SymPy Number? Please add a test to printing/tests/test_precedence.py (or similar) that demonstrates correct behavior.